os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -193,6 +196,24 @@ static int binary_manager_load(int bin_idx)
 		if (!binp)
 #endif
 		{
+#ifdef CONFIG_BINARY_SIGNING
+			/* Check signature */
+			ret = up_verify_usersignature(BIN_PARTADDR(bin_idx, (BIN_USEIDX(bin_idx))));
```

Comment:
Now, `up_verify_usersignature` always returns `OK`.
But you checked the return is `OK` or not.
Could you check this?

Reply:
Yes, I added it for code sync with internal codes although we support make binary signing in public now.

Reply:
You mean you will add the return condition in `up_verify_usersignature` later?